dvc/tree/base.py

```diff
@@ -404,7 +408,7 @@ def _download_dir(
                     dir_mode=dir_mode,
                 )
             )
-            with ThreadPoolExecutor(max_workers=self.JOBS) as executor:
```

Comment:
So now `self.JOBS`( a default number of jobs provided by each tree) is not used at all. This doesn't look right.

Reply:
@I159 Take a look at `hash_jobs` property, you need to do pretty much the same thing.

Reply:
Yep, I reached this idea. 🙏🏼